tests/dtm/test_single_process_restart.py

```diff
@@ -347,3 +359,145 @@ def test_bkt_creation_ios_after_m0d_restart(self):
+        if proc_read_op.is_alive():
+            proc_read_op.join()
+        resp = que.get()
+        assert_utils.assert_true(resp[0], resp[1])
+        workload_info = resp[1]
+
+        self.log.info("Step 6: Perform Delete Operations on data written in Step 1:")
+        self.dtm_obj.perform_ops(workload_info, que, True, True, False)
+        resp = que.get()
+        assert_utils.assert_true(resp[0], resp[1])
+
+        self.test_completed = True
+        self.log.info("ENDED: Verify continuous READ during m0d restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-41226")
+    def test_continuous_write_during_m0d_restart(self):
+        """Verify continuous WRITE during m0d restart using pkill."""
+        self.log.info("STARTED: Verify continuous WRITE during m0d restart using pkill")
+        bucket_name = 'bucket-test-41226'
+        object_prefix = 'object-test-41226'
+        log_file_prefix = 'test-41226'
+        que = multiprocessing.Queue()
+
+        self.log.info("Step 1: Start write Operations in loop:")
+        proc_write_op = multiprocessing.Process(target=self.dtm_obj.perform_write_op,
+                                                args=(bucket_name, object_prefix,
+                                                      self.test_cfg['clients'],
+                                                      self.test_cfg['samples'],
+                                                      self.test_cfg['size'],
+                                                      log_file_prefix,
+                                                      que, 10))
+        proc_write_op.start()
+
+        self.log.info("Step 3 : Perform Single m0d Process Restart During Read Operations")
+        resp = self.dtm_obj.process_restart(master_node=self.master_node_list[0],
+                                            health_obj=self.health_obj,
+                                            pod_prefix=POD_NAME_PREFIX,
+                                            container_prefix=MOTR_CONTAINER_PREFIX,
+                                            process=self.m0d_process, check_proc_state=True)
+        assert_utils.assert_true(resp, "Failure in observed during process restart/recovery")
+
+        self.log.info("Step 4: Wait for Write Operation to complete.")
+        if proc_write_op.is_alive():
+            proc_write_op.join()
+        resp = que.get()
+        assert_utils.assert_true(resp[0], resp[1])
+        workload_info = resp[1]
+
+        self.log.info("Step 5: Perform Validate and Delete Operations on data written in Step 1:")
+        self.dtm_obj.perform_ops(workload_info, que, True, True, False)
+        resp = que.get()
+        assert_utils.assert_true(resp[0], resp[1])
+
+        self.test_completed = True
+        self.log.info("ENDED: Verify continuous WRITE during m0d restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-41228")
```

Comment:
what's the diff between 41223 and this test?

Reply:
41223 is for object delete for single bucket. 41228 is object delete for multiple bucket (i.e loop)